recipes/openvr/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class OpenvrConan(ConanFile):
+    name = "openvr"
+    description = "API and runtime that allows access to VR hardware from applications have specific knowledge of the hardware they are targeting."
+    topics = ("conan", "openvr", "vr", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ValveSoftware/openvr"
+    license = "BSD 3-Clause"
+    exports_sources = "CMakeLists.txt",
+    generators = "cmake"
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows" or self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "{}-{}".format(self.name, self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["BUILD_SHARED"] = self.options.shared
+        cmake.definitions["USE_LIBCXX"] = self.settings.compiler != "Visual Studio" and str(self.settings.compiler.libcxx) == "libc++"
```

Comment:
```suggestion
        cmake.definitions["USE_LIBCXX"] = str(self.settings.compiler.libcxx) == "libc++"
```

Visual Studio doesn't use libc++